pandas/tests/indexes/multi/test_format.py

```diff
@@ -126,3 +85,133 @@ def test_repr_max_seq_item_setting(idx):
     with pd.option_context("display.max_seq_items", None):
         repr(idx)
         assert '...' not in str(idx)
+
+
+@pytest.mark.skipif(PY2, reason="repr output is different for python2")
```

Comment:
can you add at least 1 test that checks PY2 as well, ideally all tests should just check PY2 as well.